### PR TITLE
Clarify the mapping of Int @default(autoincrement()) on postgres

### PR DESCRIPTION
**Pull request by tomhoule**
We now match all sizes of serials, with native types both on and off.

### CODE REVIEW

File: libs/datamodel/connectors/dml/src/default_value.rs [hunk 1]
```diff
@@ -23,6 +23,11 @@ impl DefaultValue {
         }
     }
 
+    /// Does this match @default(autoincrement())?
```
Comment (do4gr) [inline libs/datamodel/connectors/dml/src/default_value.rs:26]: the test below answers that right?

Comment (tomhoule) [reply libs/datamodel/connectors/dml/src/default_value.rs:26]: Ah, it was meant as way to document the function :p the returned boolean is the answer. I should maybe formulate that differently.

**Review by do4gr — APPROVED** [review]
LGTM

File: libs/datamodel/connectors/dml/src/default_value.rs (hunk 1 above)
Comment (do4gr) [reply libs/datamodel/connectors/dml/src/default_value.rs:26]: Ah ok

**Pull request merged**